src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java

```diff
@@ -11,17 +11,14 @@
 
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
 
-import javax.swing.JPanel;
-import javax.swing.JComboBox;
-import javax.swing.JCheckBox;
+import javax.swing.*;
 
-import java.awt.Font;
-import java.awt.Dimension;
-import java.awt.GridLayout;
+import java.awt.*;
```

Comment:
Specify the import library, change the "*"

Reply:
ok